src/Tablet/Sensors/SensorsTab.module.css

```diff
@@ -1,3 +1,16 @@
-.column {
+.gridContainer {
   margin-top: 20px;
+  display: grid;
+  grid-template-columns: auto auto auto;
```

Comment:
`grid-template-columns: 1fr 1fr 1fr` looks much better to m e as the columns will be equal in size

![image](https://user-images.githubusercontent.com/733074/61362049-d1771280-a881-11e9-8076-f764a28e234c.png)

Reply:
i second that